lib/Tool/Session.php

```diff
@@ -11,97 +11,37 @@
  * Full copyright and license information is available in
  * LICENSE.md which is distributed with this source code.
  *
- *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
- *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
 namespace Pimcore\Tool;
 
-use Pimcore\Bundle\AdminBundle\Session\Handler\AdminSessionHandler;
-use Pimcore\Bundle\AdminBundle\Session\Handler\AdminSessionHandlerInterface;
-use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Session\Attribute\AttributeBagInterface;
+use Symfony\Component\HttpFoundation\Session\SessionBagInterface;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
 
 final class Session
```

Comment:
@dpfaffenbauer I'm just wondering whether we should really keep this helper class or if we should kick it ... there're no real benefit anymore, right?

Reply:
Since the `getBag` from `SessionInterface` doesn't necessarily return a `AttributeBagInterface`, I thought it is a nice helper to not need to check in every place if the right type is returned.

Reply:
👍 fine for me, as it also requires less changes in all the other files as a result, but maybe we should tag the class then as `@internal` - then we're flexible in future 😊

Reply:
absolutely